src/main/bash/sdkman-help.sh

```diff
@@ -28,7 +28,7 @@ function __sdk_help {
 	echo "       use       or u    <candidate> [version]"
 	echo "       default   or d    <candidate> [version]"
 	echo "       current   or c    [candidate]"
-	echo "       outdated  or o    [candidate]"
+	echo "       upgrade   or g    [candidate]"
```

Comment:
`g` probably doesn't make sense for me as mnemonic. I would probably prefer `ug`.